File: Reverse.py
def reverseQueue(self):
    if self.is_empty():
        return
    left = self.front
    right = self.rear
    while left < right:
        self.array[left], self.array[right] = self.array[right], self.array[left]
        left += 1
        right -= 1

File: C++/Reverse.cpp
void reverseQueue() {
    if (isEmpty()) {
        return;
    }
    int left = front;
    int right = rear;
    while (left < right) {
        int temp = array[left];
        array[left] = array[right];
        array[right] = temp;
        left++;
        right--;
    }
}
